platformio.ini

```diff
@@ -30,12 +30,15 @@ lib_deps =
   https://github.com/teemuatlut/TMC2208Stepper/archive/v0.0.3.zip
   Adafruit NeoPixel@1.1.3
   https://github.com/lincomatic/LiquidTWI2/archive/master.zip
-  https://github.com/trinamic/TMC26XStepper/archive/master.zip
+#  https://github.com/trinamic/TMC26XStepper/archive/master.zip
+  https://github.com/trinamic/TMC26XStepper.git
```

Comment:
How can this be getting something different from `master.zip`?

It would be ideal if we can avoid having a dependency on `git` as this is frustrating for n00bs.

Reply:
So, I just did a compare on them and they are identical. What's up with the `_BV` missing issue, then?